packages/ibm-products/src/components/Datagrid/Datagrid/addons/stateReducer.js

```diff
@@ -114,7 +114,10 @@ export const stateReducer = (newState, action) => {
         const newData = { ...newState.selectedRowData };
         const dataWithRemovedRow = Object.fromEntries(
           Object.entries(newData).filter(([key]) => {
-            return parseInt(key) !== parseInt(getRowId(rowData.original, rowData.index));
+            return (
+              parseInt(key) !==
+              parseInt(getRowId(rowData.original, rowData.index))
+            );
           })
```

Comment:
I'm not seeing any real changes. It looks like you saved this page, and it reformatted it for you.